### PR TITLE
[New] Drush Drupal distro x

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- Created new install Drush guides for (Centos, Debian, and Ubuntu)
- Created new install Drupal with Drush for (Centos, Debian, and Ubuntu)
- Created new install Themes and Modules with Drush for (Centos, Debian, and Ubuntu)
- Added a deprecated note to the existing themes and modules on Debian 7 guide.

PS: Drush 9 is used in all these guides

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 7c3de1a319ccc3c18a85aaad7fec98c0feff5501

https://deploy-preview-3160--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-centos-8/index.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        drush --version
+
+    You should see a similar output:
+
+    {{< output >}}
+Drush Commandline Tool 10.2.2
+    {{</ output >}}
+
+    {{< note >}}
+If your system is having trouble finding the `drush` command, add the `vendor` directory to your `$PATH`. using the text editor of your choice, edit your `.bashrc` file to add the directory to it's path:
```
Comment (andystevensname) [inline docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-centos-8/index.md]: Capitalize Using

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+        curl -sS https://getcomposer.org/installer | php
+
+1.  Move the composer.phar file to `/usr/local/bin/`, so that it can be accessed from any directory:
+
+        sudo mv composer.phar /usr/local/bin/composer
+
+1. Composer has a few PHP dependencies it needs in order to run. Install them on your Ubuntu system:
+
+        sudo apt-get install php-xml, zip
```
Comment (andystevensname) [inline docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md]: Is the comma here going to result in an error?

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-debian-10/index.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+1.  Run the `source` command on the `.bashrc` file to enable the changes:
+
+        source ~/.bashrc
+
+1. Check to see that Drush was installed successfully:
+
+        drush --version
+
+    {{< note >}}
+You can install the [Drush Launcher](https://github.com/drush-ops/drush-launcher) a utility to be able to call Drush globally. This program listens on your `$PATH` and hands control to a site-local Drush that is in the `~/vendor` directory.
```
Comment (andystevensname) [inline docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-debian-10/index.md]: Comma after Drush Launcher

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+1.  Run the `source` command on the `.bashrc` file to enable the changes:
+
+        source ~/.bashrc
+
+1. Check to see that Drush was installed successfully:
+
+        drush --version
+
+    {{< note >}}
+You can install the [Drush Launcher](https://github.com/drush-ops/drush-launcher) a utility to be able to call Drush globally. This program listens on your `$PATH` and hands control to a site-local Drush that is in the `~/vendor` directory.
```
Comment (andystevensname) [inline docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md]: Comma after Drush Launcher

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md (hunk 2 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md]: no

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md (hunk 4 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04/index.md]: done

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-debian-10/index.md (hunk 3 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-debian-10/index.md]: done

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-centos-8/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-centos-8/index.md]: done

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-ubuntu-18-04/index.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+1.  Extract the downloaded tarball's contents into your site's document root:
+
+        sudo tar -zxvf drupal-8.*.tar.gz --strip-components=1 -C public_html
+
+1.  Drupal depends on a PHP graphics library called GD. Install GD and other dependencies:
+
+        sudo apt-get install php-gd php-xml php-dom php-simplexml php-mbstring
+
+1. Create your Drupal 8 installation's `settings.php` file by creating a copy of the default settings file. This file will be configured when you run through Drupal's web configuration in the [Drupal First Start](#drupal-first-start) section.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-ubuntu-18-04/index.md]: done

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-debian-10/index.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+1.  Extract the downloaded tarball's contents into your site's document root:
+
+        sudo tar -zxvf drupal-8.*.tar.gz --strip-components=1 -C public_html
+
+1.  Drupal depends on a PHP graphics library called GD. Install GD and other dependencies:
+
+        sudo apt-get install php-gd php-xml php-dom php-simplexml php-mbstring
+
+1. Create your Drupal 8 installation's `settings.php` file by creating a copy of the default settings file. This file will be configured when you run through Drupal's web configuration in the [Drupal First Start](#drupal-first-start) section.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-debian-10/index.md]: done

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-centos-8/index.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+
+1.  Extract the downloaded tarball's contents into your site's document root:
+
+        sudo tar -zxvf drupal-8.*.tar.gz --strip-components=1 -C public_html
+
+1.  Drupal depends on a PHP graphics library called GD. Install GD and other dependencies:
+
+        sudo yum install -y php php-{cli,mysqlnd,json,opcache,xml,mbstring,gd,curl}
+
+1. Create your Drupal 8 installation's `settings.php` file from the default settings file. This file will be configured when you run through Drupal's web configuration in the [Drupal First Start](#drupal-first-start) section.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-centos-8/index.md]: done

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-centos-8/index.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+
+### Setting the Site's Ownership and Permissions
+
+In server administration, there are many options for user and group permissions. The directions below create a site owner and a site owner's group. The site owner will be added to the Apache web server's group, named `apache`. Then, read, write, and execute permissions are granted to the `apache` user and group.
+
+1. To create a new user for the site owner position, see the [Add a Limited User Account](/docs/security/securing-your-server/#add-a-limited-user-account) section of the [Securing Your Server](/docs/security/securing-your-server/) guide.
+
+1.  From the `public_html` directory, change ownership of the site to the site owner and group. Replace `example_user` below with the chosen owner's username:
+
+        sudo chown -R apache:apache sites/default
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-centos-8/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-ubuntu-18-04/index.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+keywords: ["drupal", "cms", "content management system", "content management framework", "drush", "ubuntu"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-02-29
+modified_by:
+    name: Linode
+published: 2020-02-29
+title: How to Install Drupal using Drush on Ubuntu 18.04
+h1_title: Install Drupal using Drush on Ubuntu 18.04
+---
+
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-ubuntu-18-04/index.md]: added

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-debian-10/index.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+keywords: ["drupal", "cms", "content management system", "content management framework","drush", "ubuntu"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-02-29
+modified_by:
+    name: Linode
+published: 2020-02-29
+title: How to Install Drupal using Drush on Debian 10
+h1_title: Install Drupal using Drush on Debian 10
+---
+
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-debian-10/index.md]: added

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-centos-8/index.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+keywords: ["drupal", "cms", "content management system", "content management framework","drush", "centos"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-02-29
+modified_by:
+    name: Linode
+published: 2020-02-29
+title: How to Install Drupal using Drush on CentOS 8
+h1_title: Install Drupal using Drush on CentOS 8
+---
+
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-using-drush-on-centos-8/index.md]: added

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-debian-10/index.md [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+    sudo chmod 774 -R /var/www/html/example.com/public_html
+    {{</ note >}}
+
+3.  Enable the theme that you downloaded in the previous step. For example, to enable the `bootstrap` theme run the following command. Replace `bootstrap` with the name of your theme.
+
+        drush theme:enable bootstrap
+
+4.  As an example, set the [Corporate Clean](https://www.drupal.org/project/corporateclean) theme as the default and active theme:
+
+        drush config-set system.theme default omega
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-debian-10/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-centos-8/index.md [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+    sudo chmod 774 -R /var/www/html/example.com/public_html
+    {{</ note >}}
+
+3.  Enable the theme that you downloaded in the previous step. For example, to enable the `bootstrap` theme run the following command. Replace `bootstrap` with the name of your theme.
+
+        drush theme:enable bootstrap
+
+4.  As an example, set the [Corporate Clean](https://www.drupal.org/project/corporateclean) theme as the default and active theme:
+
+        drush config-set system.theme default omega
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-centos-8/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-ubuntu-18-04/index.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+    sudo chmod 774 -R /var/www/html/example.com/public_html
+    {{</ note >}}
+
+3.  Enable the theme that you downloaded in the previous step. For example, to enable the `bootstrap` theme run the following command. Replace `bootstrap` with the name of your theme.
+
+        drush theme:enable bootstrap
+
+4.  As an example, set the [Corporate Clean](https://www.drupal.org/project/corporateclean) theme as the default and active theme:
+
+        drush config-set system.theme default omega
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-ubuntu-18-04/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-ubuntu-18-04/index.md [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+
+{{< note >}}
+The Drush commands to download or enable themes and modules vary depending on the version of Drush that you have installed. This guide uses Drush 10.
+{{< /note >}}
+
+## Installing Themes with Drush
+
+In this section you will download, enable, and set a Drupal theme using Drush.
+
+1. Navigate to [Drupal's Download and Extend page](https://www.drupal.org/project/project_theme) to find the theme you want to download. The drush download name is usually appended to the end of the Drupal theme page's URL; for example, `drupal.org/project/project/my_theme`.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-ubuntu-18-04/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-centos-8/index.md [hunk 16]
```diff
@@ -0,0 +1,112 @@
...
+
+{{< note >}}
+The Drush commands to download or enable themes and modules vary depending on the version of Drush that you have installed. This guide uses Drush 10.
+{{< /note >}}
+
+## Installing Themes with Drush
+
+In this section you will download, enable, and set a Drupal theme using Drush.
+
+1. Navigate to [Drupal's Download and Extend page](https://www.drupal.org/project/project_theme) to find the theme you want to download. The drush download name is usually appended to the end of the Drupal theme page's URL; for example, `drupal.org/project/project/my_theme`.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-centos-8/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-debian-10/index.md [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+
+{{< note >}}
+The Drush commands to download or enable themes and modules vary depending on the version of Drush that you have installed. This guide uses Drush 10.
+{{< /note >}}
+
+## Installing Themes with Drush
+
+In this section you will download, enable, and set a Drupal theme using Drush.
+
+1. Navigate to [Drupal's Download and Extend page](https://www.drupal.org/project/project_theme) to find the theme you want to download. The drush download name is usually appended to the end of the Drupal theme page's URL; for example, `drupal.org/project/project/my_theme`.
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-debian-10/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-ubuntu-18-04/index.md [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+
+4.  Install [Composer and Drush on Ubuntu 18.04](/docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-ubuntu-18-04)
+
+
+5.  Make sure that your system is up to date, using:
+
+        sudo apt-get update && sudo apt-get upgrade
+
+
+{{< note >}}
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-ubuntu-18-04/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-debian-10/index.md [hunk 19]
```diff
@@ -0,0 +1,115 @@
...
+
+4.  Install [Composer and Drush on Debian 10](/docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-debian-10)
+
+
+5.  Make sure that your system is up to date, using:
+
+        sudo apt-get update && sudo apt-get upgrade
+
+
+{{< note >}}
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-debian-10/index.md]: fixed

File: docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-centos-8/index.md [hunk 20]
```diff
@@ -0,0 +1,112 @@
...
+1.  Install and configure a [LAMP stack on CentOS 8](/docs/web-servers/lamp/how-to-install-a-lamp-stack-on-centos-8)
+
+1.  Install [Composer and Drush on CentOS 8](/docs/websites/cms/drupal/drush-drupal/how-to-install-drush-on-centos-8)
+
+1.  Make sure that your system is up to date, using:
+
+        sudo yum update
+
+
+{{< note >}}
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/websites/cms/drupal/drush-drupal/how-to-install-drupal-themes-and-modules-using-drush-on-centos-8/index.md]: fixed

**Pull request merged**